lib/transproc/hash.rb

```diff
@@ -333,6 +333,23 @@ def self.unwrap(hash, root, keys = nil, prefix: false)
       unwrap!(copy, root, keys, prefix: prefix)
     end
 
+    # Same as `:unwrap` but handles nil values
+    #
+    # If value for a specified key is nil, or there is no such key in a hash
+    # `:try_unwrap` returns hash without the specified key and with no added element
+    #
+    # @see HashTransformations.unwrap
+    #
+    # @api public
+    def self.try_unwrap(hash, root, keys = nil, prefix: false)
```

Comment:
Could we also add a `try_unwrap!` mutating version? We have it as standard for all other hash transformations, I know @solnic mentioned we were looking at getting rid of them but I think it's best to be consistent until we do